projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="quantity" *ngIf="showQuantity">
+<div class="quantity" *ngIf="isLoaded && showQuantity">
```

Comment:
Because showing the quantity depends on the product data loading, can we wait for `productCode && showQuantity` instead and remove the extra `isLoaded` flag?

Reply:
https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/3529